homeassistant/components/google_assistant/__init__.py

```diff
@@ -36,10 +40,15 @@
             vol.Required(CONF_ACCESS_TOKEN): cv.string,
             vol.Optional(CONF_EXPOSE_BY_DEFAULT): cv.boolean,
             vol.Optional(CONF_EXPOSED_DOMAINS): cv.ensure_list,
+            vol.Required(CONF_AGENT_USER_ID): cv.string,
```

Comment:
Hi @philk, I was a bit unsure about this. 

If HA were to ever become an official Google Assistant app, then the userId will have to be unique across all "cloud" users. Asking users to input their email address here may provide a bit of leeway in that regard. 

Let me know your thoughts.

Reply:
Yup, that's kind of why I think it should just be a default for now. If/when that happens it could be made required or the default modified based on some part of the cloud auth system.